src/test/java/inputcommands/AddDeadlineCommandTest.java

```diff
@@ -0,0 +1,51 @@
+package inputcommands;
+
+import snomexceptions.InvalidCommandDateFormatException;
+import snomexceptions.InvalidCommandTaskDescException;
+import snomtasklist.TaskList;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class AddDeadlineCommandTest {
+
+    @Test
+    public void AddDeadlineCommandTest1() {
```

Comment:
Would it be better to use underscores in test method names? You could follow this format `featureUnderTest_testScenario_expectedBehavior()`